resources/qml/delegates/TextMessage.qml

```diff
@@ -33,11 +35,12 @@ MatrixText {
         padding: ` + Math.ceil(fontMetrics.lineSpacing/2) + `px;
     }
     blockquote { margin-left: 1em; }
-    ` + (!Settings.mobileMode ? `span[data-mx-spoiler] {
+    span[data-mx-spoiler] {` + (!showSpoilers ? `
         color: transparent;
-        background-color: ` + palette.text + `;
-    }` : "") +  // TODO(Nico): Figure out how to support mobile
-    `</style>
+        background-color: ` + palette.text + `;` : `
+        background-color: ` + palette.alternateBase + ';') + `
```

Comment:
Why are you setting the background twice?

Reply:
I'm not. That's a ternary operator statement.